pkg/compressor/types.go

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) 2020 SAP SE or an SAP affiliate company. All rights reserved. This file is licensed under the Apache Software License, v. 2 except as noted otherwise in the LICENSE file.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package compressor
+
+const (
+
+	// GzipCompressionPolicy is constant for gzip compression algorithm.
+	GzipCompressionPolicy = "gzip"
+	// LzwCompressionPolicy is constant for lzw compression algorithm.
+	LzwCompressionPolicy = "lzw"
+	// ZlibCompressionPolicy is constant for zlib compression algorithm.
+	ZlibCompressionPolicy = "zlib"
+
+	// DefaultCompression is constant used for whether to compress the snapshots or not.
+	DefaultCompression = true
```

Comment:
I think it will not break the current behaviour as it also supports the backward compatibility, we still can restore from the uncompressed snapshots,  but still I can make `DefaultCompression` to `false`, no issues.
CC @amshuman-kr

Reply:
Generally, keeping things backward compatible by default is a good principle. In this case, the default behaviour for the compression as `false` makes sense in that context. We will have other levels to _default_ this to `true` (say, in the gardener integration PR).